README.md

```diff
@@ -25,7 +25,7 @@ c-lightning is a lightweight, highly customizable and [standard compliant][std]
 [![Documentation Status](https://readthedocs.org/projects/lightning/badge/?version=docs)][docs]
 
 This implementation has been in production use on the Bitcoin mainnet since early 2018, with the launch of the [Blockstream Store][blockstream-store-blog].
-We recommend getting started by experimenting on `testnet` (or `regtest`), but the implementation is considered stable and can be safely used on mainnet.
+We recommend getting started by experimenting on `testnet` (or `regtest`), but this implementation is considered stable and can be safely used on mainnet.
```

Comment:
why we need to switch from `the` to `this`? IMO, `this` assumes that there are other implementations not stable, but the doc tells us only that the c-lightning can be considered stable.